### DISCUSSION TITLE
“功能”和“设置”按钮如何居右显示？

### DISCUSSION THREAD

**Discussion by CoucouEureka** [the1812/Bilibili-Evolved#3567] [提问 / Q&A] [answered]
请问如何将侧面的两个按钮放到右侧呢？

**Comment by Geeyun-JY3** [comment, marked as answer]
通用设置 > 设置面板停靠 > 右侧 
![image](https://user-images.githubusercontent.com/49396421/183040272-10399c1b-0832-4537-a970-4353d61d4c06.png)
